scrapers/nus-v2/src/services/logger/SentryStream.ts

```diff
@@ -71,20 +71,22 @@ export default function getSentryStream(config: StreamConfig = {}): Stream {
 
   const stream: any = {
     write(record: BunyanRecord) {
-      const tags = pick(record, tagProps);
-      const extra = pick(record, extraProps);
-      const level = getSentryLevel(record);
-
       Sentry.withScope((scope) => {
-        scope.setLevel(level);
-
-        each(tags, (value, prop) => {
-          scope.setTag(prop, String(value));
-        });
-
-        each(extra, (value, prop) => {
-          scope.setExtra(prop, value || null);
-        });
+        scope.setLevel(getSentryLevel(record));
+
+        scope.setTags(
+          chain(record as { [key: string]: unknown })
+            .pick(tagProps)
+            .mapValues((v) => String(v))
+            .value(),
+        );
+
+        scope.setExtras(
+          chain(record as { [key: string]: unknown })
+            .pick(extraProps)
+            .mapValues((v) => v ?? null)
+            .value(),
+        );
```

Comment:
The typings for `_.chain` is pretty gnarly. I think it's better to just use the plain versions of these, especially since it's just two calls 

```suggestion
        scope.setTags(
          mapValues(
            pick(record as { [key: string]: unknown }, tagProps),
            (v) => String(v),
          ),
        );

        scope.setExtras(
          mapValues(
            pick(record as { [key: string]: unknown }, extraProps),
            (v) => v ?? null,
          ),
        );
```